src/plone/app/mosaic/browser/main_template.py

```diff
@@ -44,7 +44,7 @@
         dummy python:plone_layout.mark_view(view);
         portal_url portal_state/portal_url;
         checkPermission nocall: context/portal_membership/checkPermission;
-        site_properties nocall: context/portal_properties/site_properties;
+        site_properties nocall: context/portal_properties/site_properties | nothing;
```

Comment:
```suggestion
```

Since we're targeting Plone 6.1+ you can remove this completely ... See https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/browser/templates/main_template.pt#L8-L19